packages/gluestick-generators/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "gluestick-generators",
+  "version": "0.0.0",
+  "description": "Package with generator API used by GlueStick",
+  "main": "src/index.js",
+  "homepage": "https://github.com/TrueCar/gluestick",
+  "bugs": "https://github.com/TrueCar/gluestick/issues",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "test-watch": "jest --watch",
+    "test-coverage": "jest --coverage",
+    "debug": "node --inspect --debug-brk ./node_modules/.bin/jest -i --env ./test/jestEnvironmentNodeDebug.js",
+    "postpublish": "./docker/create-base-image.js",
+    "update-version": "npm version",
+    "postupdate-version": "./docker/generate-dockerfile.js"
+  },
+  "keywords": [
+    "react",
+    "redux",
+    "jest",
+    "es6",
+    "es2015",
+    "universal",
+    "isomorphic",
+    "generator",
+    "boilerplate"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:TrueCar/gluestick.git"
+  },
+  "author": "Todd Williams <toddsurfs@icloud.com>",
+  "dependencies": {
+    "babel-cli": "6.22.2",
+    "babel-core": "6.22.1",
+    "babel-plugin-transform-decorators-legacy": "1.3.4",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-polyfill": "6.22.0",
+    "babel-preset-es2015": "6.22.0",
+    "babel-preset-react": "6.22.0",
+    "babel-preset-stage-0": "6.22.0",
+    "jest": "19.0.0",
+    "mkdirp": "^0.5.1"
+  },
+  "devDependencies": {
+    "temp": "0.8.3"
```

Comment:
Yea this is not needed. It is only used in `destroy.test.js` from `gluestick` package and it will likely go away from there soon too.